### PR TITLE
ports cult of st. credit

### PR DESCRIPTION
**Pull request by boodaliboo**
########################################
########################################
https://github.com/tgstation/tgstation/pull/52132 original pr
########################################
########################################

### Intent of your Pull Request
adds a new capitalist religion that earns points through money allowing them to get a cool fancy suit and monicle when enough money is reached
#### Changelog

:cl:  
rscadd: Adds a new religion
/:cl:

### CODE REVIEW

**Comment by Xoxeyos** [comment]
Time to L. Ron B Hubbard myself, build a megachurch, and make the station pay for it.

**Comment by Hopekz** [comment]
Cool Scientology in ss13

**Comment by boodaliboo** [comment]
opinions on buffing the rite rewards?

**Comment by fluffe9911** [comment]
#Bringthegolemback

**Comment by boodaliboo** [comment]
> #Bringthegolemback

@alexkar598 would allowing a capitalist golem  to be created through religion for a insane ammount of money be okay. im thinking like 1 million dollars worth to become one

**Comment by jcatjmeow** [comment]
Capitalist golem is a conversion antag. So, absolutely not

**Comment by boodaliboo** [comment]
> Capitalist golem is a conversion antag. So, absolutely not

what do you think a reasonable reward could be? becoming gold golems?

**Comment by fluffe9911** [comment]
or instead of being dumb about it you could make a variant of the capitalist golem that DOESNT convert like the original pr author did before he removed the golem and it got closed

**Comment by boodaliboo** [comment]
> or instead of being dumb about it you could make a variant of the capitalist golem that DOESNT convert like the original pr author did before he removed the golem and it got closed

later

File: code/modules/religion/religion_sects.dm [hunk 1]
```diff
@@ -175,3 +175,62 @@
 	to_chat(L, "<span class='notice'>You offer [the_cell]'s power to [GLOB.deity], pleasing them.</span>")
 	qdel(I)
 	return TRUE
+// cool new religion thats based off of money. (am i doing autodoc correctly?)
```
Comment (alexkar598) [inline code/modules/religion/religion_sects.dm]: No you arent, read the guide

File: code/modules/religion/religion_sects.dm [hunk 2]
```diff
@@ -175,3 +175,62 @@
...
+	var/mob/living/carbon/human/H = L
+	var/obj/item/card/id/id_card = H.get_idcard()
+	var/obj/item/card/id/id_cardu = user.get_idcard()
+	var/money_check = 500
+
+	if(!id_card.registered_account.account_balance > money_check)
+		user.visible_message("<span class='notice'>[H] is too poor to recieve [GLOB.deity]'s blessing!</span>")
+	else
+		var/heal_amt = 10
+		var/list/hurt_limbs = H.get_damaged_bodyparts(1, 1, null, BODYPART_ORGANIC)
```
Comment (alexkar598) [inline code/modules/religion/religion_sects.dm]: Use TRUE instead of 1

File: code/modules/religion/rites.dm [hunk 3]
```diff
@@ -101,3 +101,19 @@
 	human2borg.set_species(/datum/species/android)
 	human2borg.visible_message("<span class='notice'>[human2borg] has been converted by the rite of [name]!</span>")
 	return TRUE
+
+/*********Capitalists**********/
+
+/datum/religion_rites/looks
```
Comment (alexkar598) [inline code/modules/religion/rites.dm:123]: Autodocs

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/modules/religion/religion_sects.dm (hunk 2 above)
Comment (boodaliboo) [reply code/modules/religion/religion_sects.dm]: i have no clue what your asking

File: code/modules/religion/rites.dm (hunk 3 above)
Comment (boodaliboo) [reply code/modules/religion/rites.dm:123]: will be done soon

File: code/modules/religion/religion_sects.dm (hunk 2 above)
Comment (alexkar598) [reply code/modules/religion/religion_sects.dm]:
```suggestion
		var/list/hurt_limbs = H.get_damaged_bodyparts(TRUE, TRUE, null, BODYPART_ORGANIC)
```

File: code/modules/religion/religion_sects.dm [hunk 4]
```diff
@@ -175,3 +175,64 @@
 	to_chat(L, "<span class='notice'>You offer [the_cell]'s power to [GLOB.deity], pleasing them.</span>")
 	qdel(I)
 	return TRUE
+ /*
+a new religious sect based around donating money for favors
+  */
```
Comment (alexkar598) [inline code/modules/religion/religion_sects.dm]: Thats not how you format it, you need to have stars on each line
```c
/*
 *  text
 *  e
 */
```

Comment (alexkar598) [reply code/modules/religion/religion_sects.dm]: Also you need to describe the religion better than this..

File: code/modules/religion/rites.dm [hunk 5]
```diff
@@ -101,3 +101,21 @@
 	human2borg.set_species(/datum/species/android)
 	human2borg.visible_message("<span class='notice'>[human2borg] has been converted by the rite of [name]!</span>")
 	return TRUE
+
+/*********Capitalists**********/
+/*
+this rites gives you some snazzy looking clothes for 100 favor
+  */
```
Comment (alexkar598) [inline code/modules/religion/rites.dm]: Invalid comment structure

**Review by alexkar598 — DISMISSED** [review]

File: code/modules/religion/religion_sects.dm (hunk 4 above)
Comment (boodaliboo) [reply code/modules/religion/religion_sects.dm]: ok

**Review by fluffe9911 — APPROVED** [review]
ok

**Pull request merged**